geode-core/src/test/java/org/apache/geode/internal/tcp/TCPConduitTest.java

```diff
@@ -72,11 +73,24 @@ public void setUp() throws Exception {
         .thenReturn(mock(DistributionManager.class));
   }
 
+  @Test
+  public void closedConduitDoesNotThrowNPEWhenAskedForBufferPool() {
+    directChannel.getDM(); // Mockito demands that this mock be used in this test
+    TCPConduit tcpConduit =
+        new TCPConduit(membership, 0, localHost, false, directChannel, mock(BufferPool.class),
+            new Properties(),
+            TCPConduit -> connectionTable, socketCreator, doNothing(), false);
+    InternalDistributedMember member = mock(InternalDistributedMember.class);
+    tcpConduit.stop(null);
+    assertThat(tcpConduit.getBufferPool()).isNotNull();
```

Comment:
and now the `tcpConduit` is constructed with a `BufferPool` this won't ever be NULL